pkg/scheduler/preemption/preemption.go

```diff
@@ -96,8 +96,24 @@ func candidatesFromCQOrUnderThreshold(candidates []*workload.Info, clusterQueue
 	return result
 }
 
+type Target struct {
+	Wl     *workload.Info
+	Reason string
+}
+
+func GetWorkloadReferences(targets []*Target) []klog.ObjectRef {
```

Comment:
It seems like overkill to have this in an exported function, when it's only used in one place. I would have left it unexported in the logging.go file.

But not a big deal.